twine: add examples (#8924)
tldr-pages/tldr: pages/common/twine.md

# twine

> Utility for publishing Python packages on PyPI.

> > More information: <https://twine.readthedocs.io>.

Reviewer: ```suggestion
> More information: <https://twine.readthedocs.io/en/stable/#commands>.
```

Scroll directly to the right section that actually gives more information

- Upload to PyPI:

`twine upload dist/*`

> - Upload to the Test PyPI [r]epository and verify things look right:

Reviewer: ```suggestion
- Upload to the Test PyPI [r]epository to verify things look right:
```

`twine upload -r testpypi dist/*`

- Upload to PyPI with a specified [u]sername and [p]assword:

`twine upload -u {{username}} -p {{password}} dist/*`

- Upload to an alternative repository URL:

`twine upload --repository-url {{repository_url}} dist/*`

> - Check that your distribution’s long description should render correctly on PyPI:

Reviewer 2: ```suggestion
- Check that your distribution's long description should render correctly on PyPI:
```

`twine check dist/*`

> - Upload with a specific pypirc configuration file (<https://packaging.python.org/en/latest/specifications/pypirc>):

Reviewer 2: No links in examples.
```suggestion
- Upload using a specific pypirc configuration file:
```
It's uploaded with the pypirc inside the packages? or it's uploaded using configurations from a pypirc file?

Author: Uploaded using configurations from a pypirc file.

> `twine upload --config-file {{config_file}} dist/*`

Reviewer 2: We try to avoid abbreviations
```suggestion
`twine upload --config-file {{configuration_file}} dist/*`
```

> - Continue uploading files if one already exists. (Only valid when uploading to PyPI. Other implementations may not support this.):

Reviewer: ```suggestion
- Continue uploading files if one already exists (only valid when uploading to PyPI):
```

`twine upload --skip-existing dist/*`

> - Display extra verbosity during upload:

Reviewer 2: ```suggestion
- Upload to PyPI showing detailed information:
```
